src/main/cpp/t07_ip.cpp

```diff
@@ -29,10 +29,50 @@
 
 #include "t07_ip.h"
 #include <iostream>
-
+#include <string>
 
 using namespace std;
 
 int t07_ip() {
+	string h, number = "";
+	bool pass = true, num = true;
+	int numofpoints = 0;
+	int numofnumbers = 0;
+	cin >> h;
+	for (int i = 0; i < h.size(); i++)
+	{
+		if (h[i] == '.' && num ==false)
+		{
+			if (stoi(number) > 255)
+			{
 
+				pass = false;
+				break;
+			}
+			if (num==false)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Condition 'num==false' is always true](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4504172)